cherab/core/model/beam/charge_exchange.pyx

```diff
@@ -188,23 +188,23 @@ cdef class BeamCXLine(BeamModel):
         :param interaction_energy: The donor-receiver interaction energy in eV/amu.
         :param donor_velocity: A Vector defining the donor particle velocity in m/s.
         :param receiver_temperature: The receiver species temperature in eV.
-        :param receiver_density: The receiver species density in m^-3
         :return: The composite charge exchange rate in W.m^3.
         """
 
         cdef:
-            double z_effective, b_field, rate, total_population, population, effective_rate
+            double z_effective, b_field, rate, total_population, population, effective_rate, ion_density
```

Comment:
a minor suggestion, this line is bit too long, please consider moving part of the variables to a new line.